source/_patterns/03-uswds/sidenav/sidenav.md

```diff
@@ -0,0 +1,9 @@
+---
+el: .usa-sidenav
+title: Side navigation
+---
+See [https://designsystem.digital.gov/components/sidenav/]() and
+[https://components.designsystem.digital.gov/components/detail/sidenav--default.html]().
+
+__Variables:__
+* label: [string] Aria label of the menu.
```

Comment:
@dcmouyard Curious why this is a variable but the rest of the content is hard-coded? If this component is meant as a demo of the sidenav, versus something that would actually be embedded in a Drupal template, I don't see an advantage to making the aria label flexible.

Reply:
@kmonahan I was originally going to make this extendable by Drupal and mimic the menu macro, but then decided just to hard-code it to save time since we’re already a bit behind in the current sprint.

Reply:
Got it. Okay, cool.